spec/views/my/_list_works.html.erb_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:work_title) { 'Work Title' }
   let(:coll_title) { 'Collection Title' }
   let(:collection) { FactoryGirl.build(:collection, id: '3197z497t', title: coll_title) }
-  let!(:work) { FactoryGirl.build(:work, id: '3197z511f', title: [work_title]) }
+  let!(:work) { FactoryGirl.create(:generic_work, id: '3197z511f', title: [work_title]) }
```

Comment:
Can we switch this back to build and stub out `persisted?: true` ?  This will make the test faster.  This is not a *big* deal, but the slowness adds up, so we should nip it in the bud.